benchmarks/benchmarks/stats.py

```diff
@@ -163,6 +163,36 @@ def setup(self):
             [self.inp[0], self.inp[1]], self.inp[2], statistic="std",
             bins=[self.subbin_x_edges, self.subbin_y_edges])
 
+    def time_binned_statistic_dd_count(self):
+        stats.binned_statistic_dd(
+            [self.inp[0], self.inp[1]], self.inp[2], statistic="count",
```

Comment:
for these b/c everything is the same except the `statistic` argument, would it make sense to use:
https://asv.readthedocs.io/en/stable/writing_benchmarks.html#parametrized-benchmarks
instead? It might be cleaner code but not sure about the actual behavior of the benchmarking, I'd guess the behavior is the same.

Reply:
Oh, very good idea. I changed it. Thanks.